.github/workflows/test.yaml

```diff
@@ -35,21 +35,21 @@ jobs:
 
       - name: Run chart-testing (lint)
         id: lint
-        uses: helm/chart-testing-action@v1.0.0
+        uses: helm/chart-testing-action@v2.0.1
```

Comment:
v2 will only install `ct` and not run it: [Upgrading from v1.x.x](https://github.com/helm/chart-testing-action#upgrading-from-v1xx)

You might want to take a look at the [Example Workflow](https://github.com/helm/chart-testing-action#example-workflow) to see what else needs changing for v2 compat.

Reply:
6509809

Reply:
Thanks!